tests/analyze.R

```diff
@@ -106,20 +105,45 @@ ProcessAll = function(ctx) {
                   proportion = rappor$proportion,
                   dist = "rappor")
 
-  # Fill in zeros for values missing in RAPPOR.  It makes the ggplot bar plot
-  # look better.
+  # Fill in zeros for values missing in RAPPOR and in actual data
```

Comment:
OK I see, how about better names for "fill" and "fpfill" ?

Maybe actual_only and rappor_only.